river/anomaly/gaussian.py

```diff
@@ -0,0 +1,73 @@
+from river import anomaly, proba
+
+
+class GaussianScorer(anomaly.base.SupervisedAnomalyDetector):
```

Comment:
> Should we think about a way to differentiate univariate and multivariate anomaly detectors?

For the moment I would say no! I would say that univariate detectors can be implemented as supervised anomaly detectors. Having them as unsupervised estimators is tricky because `x` would have to be a single value rather than a `dict`.